### DISCUSSION TITLE
Crossfade

### DISCUSSION THREAD

**Discussion by Aussie-boii** [fast4x/RiMusic#2854] [Ideas]
Hey it's me again I wanted to ask if you fixed the issue of crossfade or you gave up on the idea

#2 Can you also make the animated album covers like apple Music or it's also impossible like spotify

**Comment by fast4x** [comment]
we are working on

**Comment by aneesh1122** [comment]
> Can you also make the animated album covers like apple Music or it's also impossible like spotify

Those animated album covers are given to apple music by artists. Unless there's an API which fetches those animated covers I don't see it happening
